Automattic/newspack-theme
# Fix group block alignments on homepage

### All Submissions:


### Changes proposed in this Pull Request:

This PR is homepage specific: if fixes some alignment weirdness when you nest blocks inside of a Group block that's set to full-width. Rather than keeping their existing widths, they match the width of the containing block.

One big use for this block, I imagine, will be creating visually distinct sections on the homepage, by giving it a background colour. So I started with a fix there; I still need to work out some width issues on single posts and regular pages before tackling it there. 

The Group block will likely be in WordPress 5.3; right now it's just available in the Gutenberg plugin.

### How to test the changes in this Pull Request:

1. Install and activate the [Gutenberg plugin](https://en-ca.wordpress.org/plugins/gutenberg/), so you have access to the Group block.
1. On a site with a static front page, add a Group block and set it to align full. Giving it a background colour will also make this a bit easier to test. 
2. Inside of that block, add a Newspack article block (or any other block).
3. Publish the post; note that in both the editor and on the front end, the block inside of the Group block is also full width.
4. Apply the PR.
5. Confirm that the block inside of the Group block now matches the site's content width.

### Other information:


**Edited to add:** Steps for how to get the Group block to test this, which are pretty dang important 🤦‍♀
